test_e2e/dest/delta-table-s3.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
If this is a new test script, can we leverage the new testing framework and add an integration test for this rather than an e2e test [here](https://github.com/Unstructured-IO/unstructured-ingest/tree/main/test/integration/connectors)

Reply:
Added the suggested integration test. The e2e test will be removed after the new test is positively reviewed